src/scripts/Utils.js

```diff
@@ -23,3 +23,12 @@ export function clearSelect(select) {
 export function getRandomInteger(min, max) {
     return Math.floor(Math.random() * (max - min)) + min;
 }
+
+export function getFirstObjectFromArray(array) {
+    for (var key in array)
```

Comment:
I think you can use something like `const [key, value] = Object.entries(array)[0];` with an additional check for entries not being empty.

Also, it seems that this function is not actually working with arrays, but with objects. Maybe rename it then to something like `getFirstKeyValueFromObject`?

Reply:
I renamed function, and used your code suggestion.